spaces/bella/skin.scss

```diff
@@ -0,0 +1,9 @@
+.bella {
+  --primary-color: #384aff;
```

Comment:
This skin seem exactly the same as the default skin. If that's the case we don't need this file in the PR, could you remove it?

Reply:
removed the skin config